src/main/kotlin/io/github/rybalkinsd/kohttp/ext/HttpContextExt.kt

```diff
@@ -14,10 +14,21 @@ fun HttpContext.url(url: URL) {
 
     host = url.host ?: throw IllegalArgumentException("unexpected host: $host")
 
-    if (url.port != -1) {
+    if (url.port != -1)
         port = url.port
-    }
+
     path = url.path
+
+    if (url.query?.isNotBlank() == true) {
```

Comment:
The condition can be simplified. `if (url.query?.isNotBlank())`

Reply:
Unfortunately, `url.query?.isNotBlank()` produces `Boolean?` which can not be used as is inside `if` statement